2025-08-07 09:19:36 GMT [7]: [3-1] user=,db=,app=,client= LOG:  starting PostgreSQL 17.5 (Debian 17.5-1.pgdg120+1) on aarch64-unknown-linux-gnu, compiled by gcc (Debian 12.2.0-14) 12.2.0, 64-bit
2025-08-07 09:19:36 GMT [7]: [4-1] user=,db=,app=,client= LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-08-07 09:19:36 GMT [7]: [5-1] user=,db=,app=,client= LOG:  listening on IPv6 address "::", port 5432
2025-08-07 09:19:36 GMT [7]: [6-1] user=,db=,app=,client= LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-08-07 09:19:36 GMT [12]: [1-1] user=,db=,app=,client= LOG:  database system was interrupted; last known up at 2025-08-07 09:17:13 GMT
2025-08-07 09:19:36 GMT [12]: [2-1] user=,db=,app=,client= LOG:  database system was not properly shut down; automatic recovery in progress
2025-08-07 09:19:36 GMT [12]: [3-1] user=,db=,app=,client= LOG:  redo starts at 0/40008D8
2025-08-07 09:19:36 GMT [12]: [4-1] user=,db=,app=,client= LOG:  invalid record length at 0/4001040: expected at least 24, got 0
2025-08-07 09:19:36 GMT [12]: [5-1] user=,db=,app=,client= LOG:  redo done at 0/4001008 system usage: CPU: user: 0.00 s, system: 0.00 s, elapsed: 0.00 s
2025-08-07 09:19:36 GMT [10]: [1-1] user=,db=,app=,client= LOG:  checkpoint starting: end-of-recovery immediate wait
2025-08-07 09:19:36 GMT [10]: [2-1] user=,db=,app=,client= LOG:  checkpoint complete: wrote 4 buffers (0.0%); 0 WAL file(s) added, 0 removed, 0 recycled; write=0.001 s, sync=0.010 s, total=0.019 s; sync files=3, longest=0.005 s, average=0.004 s; distance=1 kB, estimate=1 kB; lsn=0/4001040, redo lsn=0/4001040
2025-08-07 09:19:36 GMT [7]: [7-1] user=,db=,app=,client= LOG:  database system is ready to accept connections
cp: cannot create regular file '/var/lib/postgresql/archive/000000010000000000000001': No such file or directory
2025-08-07 09:19:36 GMT [15]: [1-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-07 09:19:36 GMT [15]: [2-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/000000010000000000000001 && cp pg_wal/000000010000000000000001 /var/lib/postgresql/archive/000000010000000000000001
cp: cannot create regular file '/var/lib/postgresql/archive/000000010000000000000001': No such file or directory
2025-08-07 09:19:37 GMT [15]: [3-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-07 09:19:37 GMT [15]: [4-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/000000010000000000000001 && cp pg_wal/000000010000000000000001 /var/lib/postgresql/archive/000000010000000000000001
cp: cannot create regular file '/var/lib/postgresql/archive/000000010000000000000001': No such file or directory
2025-08-07 09:19:38 GMT [15]: [5-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-07 09:19:38 GMT [15]: [6-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/000000010000000000000001 && cp pg_wal/000000010000000000000001 /var/lib/postgresql/archive/000000010000000000000001
2025-08-07 09:19:38 GMT [15]: [7-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "000000010000000000000001" failed too many times, will try again later
2025-08-07 09:19:46 GMT [24]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 09:19:46 GMT [24]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 09:19:46 GMT [24]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=psql
2025-08-07 09:19:46 GMT [24]: [4-1] user=postgres,db=postgres,app=psql,client=[local] ERROR:  database "testdb" already exists
2025-08-07 09:19:46 GMT [24]: [5-1] user=postgres,db=postgres,app=psql,client=[local] STATEMENT:  CREATE DATABASE testdb
2025-08-07 09:19:46 GMT [24]: [6-1] user=postgres,db=postgres,app=psql,client=[local] LOG:  disconnection: session time: 0:00:00.005 user=postgres database=postgres host=[local]
2025-08-07 09:19:46 GMT [26]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 09:19:46 GMT [26]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 09:19:46 GMT [26]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=psql
2025-08-07 09:19:46 GMT [26]: [4-1] user=postgres,db=postgres,app=psql,client=[local] ERROR:  role "replicator" already exists
2025-08-07 09:19:46 GMT [26]: [5-1] user=postgres,db=postgres,app=psql,client=[local] STATEMENT:  CREATE USER replicator WITH REPLICATION LOGIN PASSWORD 'replicator'
2025-08-07 09:19:46 GMT [26]: [6-1] user=postgres,db=postgres,app=psql,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-07 09:19:46 GMT [28]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 09:19:46 GMT [28]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 09:19:46 GMT [28]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=psql
2025-08-07 09:19:46 GMT [28]: [4-1] user=postgres,db=postgres,app=psql,client=[local] ERROR:  role "efm" already exists
2025-08-07 09:19:46 GMT [28]: [5-1] user=postgres,db=postgres,app=psql,client=[local] STATEMENT:  CREATE USER efm WITH LOGIN PASSWORD 'efm'
2025-08-07 09:19:46 GMT [28]: [6-1] user=postgres,db=postgres,app=psql,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
2025-08-07 09:19:46 GMT [30]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 09:19:46 GMT [30]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 09:19:46 GMT [30]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=psql
2025-08-07 09:19:46 GMT [30]: [4-1] user=postgres,db=postgres,app=psql,client=[local] LOG:  disconnection: session time: 0:00:00.008 user=postgres database=postgres host=[local]
2025-08-07 09:19:47 GMT [162]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=172.20.0.10 LOG:  connection received: host=172.20.0.10 port=49788
2025-08-07 09:19:47 GMT [162]: [2-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authenticated: identity="efm" method=md5 (/var/lib/postgresql/data/pg_hba.conf:15)
2025-08-07 09:19:47 GMT [162]: [3-1] user=efm,db=postgres,app=[unknown],client=172.20.0.10 LOG:  connection authorized: user=efm database=postgres
2025-08-07 09:19:47 GMT [162]: [4-1] user=efm,db=postgres,app=efm-5.0,client=172.20.0.10 LOG:  disconnection: session time: 0:00:00.039 user=efm database=postgres host=172.20.0.10 port=49788
2025-08-07 09:20:06 GMT [175]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 09:20:06 GMT [175]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 09:20:06 GMT [175]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-07 09:20:06 GMT [175]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.000 user=postgres database=postgres host=[local]
2025-08-07 09:20:26 GMT [272]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 09:20:26 GMT [272]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 09:20:26 GMT [272]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=psql
2025-08-07 09:20:26 GMT [272]: [4-1] user=postgres,db=postgres,app=psql,client=[local] LOG:  disconnection: session time: 0:00:00.003 user=postgres database=postgres host=[local]
2025-08-07 09:20:26 GMT [279]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=::1 LOG:  connection received: host=::1 port=52182
2025-08-07 09:20:26 GMT [286]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 09:20:26 GMT [286]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 09:20:26 GMT [286]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=psql
2025-08-07 09:20:26 GMT [286]: [4-1] user=postgres,db=postgres,app=psql,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
2025-08-07 09:20:26 GMT [293]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 09:20:26 GMT [293]: [2-1] user=postgres,db=testdb,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 09:20:26 GMT [293]: [3-1] user=postgres,db=testdb,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=testdb application_name=psql
2025-08-07 09:20:26 GMT [293]: [4-1] user=postgres,db=testdb,app=psql,client=[local] ERROR:  relation "test_replication" does not exist at character 37
2025-08-07 09:20:26 GMT [293]: [5-1] user=postgres,db=testdb,app=psql,client=[local] STATEMENT:  SELECT id, message, created_at FROM test_replication ORDER BY created_at DESC LIMIT 3;
2025-08-07 09:20:26 GMT [293]: [6-1] user=postgres,db=testdb,app=psql,client=[local] LOG:  disconnection: session time: 0:00:00.005 user=postgres database=testdb host=[local]
2025-08-07 09:20:36 GMT [301]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 09:20:36 GMT [301]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 09:20:36 GMT [301]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=pg_isready
2025-08-07 09:20:36 GMT [301]: [4-1] user=postgres,db=postgres,app=pg_isready,client=[local] LOG:  disconnection: session time: 0:00:00.001 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/000000010000000000000001': No such file or directory
2025-08-07 09:20:38 GMT [15]: [8-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-07 09:20:38 GMT [15]: [9-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/000000010000000000000001 && cp pg_wal/000000010000000000000001 /var/lib/postgresql/archive/000000010000000000000001
cp: cannot create regular file '/var/lib/postgresql/archive/000000010000000000000001': No such file or directory
2025-08-07 09:20:39 GMT [15]: [10-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-07 09:20:39 GMT [15]: [11-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/000000010000000000000001 && cp pg_wal/000000010000000000000001 /var/lib/postgresql/archive/000000010000000000000001
2025-08-07 09:20:40 GMT [313]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 09:20:40 GMT [313]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 09:20:40 GMT [313]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=psql
2025-08-07 09:20:40 GMT [313]: [4-1] user=postgres,db=postgres,app=psql,client=[local] LOG:  disconnection: session time: 0:00:00.002 user=postgres database=postgres host=[local]
cp: cannot create regular file '/var/lib/postgresql/archive/000000010000000000000001': No such file or directory
2025-08-07 09:20:40 GMT [15]: [12-1] user=,db=,app=,client= LOG:  archive command failed with exit code 1
2025-08-07 09:20:40 GMT [15]: [13-1] user=,db=,app=,client= DETAIL:  The failed archive command was: test ! -f /var/lib/postgresql/archive/000000010000000000000001 && cp pg_wal/000000010000000000000001 /var/lib/postgresql/archive/000000010000000000000001
2025-08-07 09:20:40 GMT [15]: [14-1] user=,db=,app=,client= WARNING:  archiving write-ahead log file "000000010000000000000001" failed too many times, will try again later
2025-08-07 09:20:50 GMT [322]: [1-1] user=[unknown],db=[unknown],app=[unknown],client=[local] LOG:  connection received: host=[local]
2025-08-07 09:20:50 GMT [322]: [2-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authenticated: identity="postgres" method=peer (/var/lib/postgresql/data/pg_hba.conf:7)
2025-08-07 09:20:50 GMT [322]: [3-1] user=postgres,db=postgres,app=[unknown],client=[local] LOG:  connection authorized: user=postgres database=postgres application_name=psql
2025-08-07 09:20:50 GMT [322]: [4-1] user=postgres,db=postgres,app=psql,client=[local] LOG:  disconnection: session time: 0:00:00.007 user=postgres database=postgres host=[local]
